content/collections/audit-works.html

```diff
@@ -1,3 +1,315 @@
+<item>
+  <field name="logo">
+    thirdweb.png
+  </field>
+  <field name="title">
+    thirdWeb-15
+  </field>
+  <field name="subtitle">
+    Web3 developer tooling
+  </field>
+  <field name="date">
+    2023-10-06
+  </field>
+  <field name="about">
```

Comment:
Audited BurnToClaimERC721 as well as changes made to MarketplaceV3 since the previous audit.